DOC-2181: add fix documentation for TINY-10285 to the 6.8 release notes. (#2977)
tinymce/tinymce-docs: modules/ROOT/pages/6.8-release-notes.adoc

=== Spellchecker does not trigger on shift+enter
// TINY-10285
Previously, while pressing **Enter**, **Spacebar** or changing context within the editor after a incorrectly spelled word would trigger the **Spell Checker Pro** plugin. Although it was discovered that after pressing **Shift+Enter**, the plugin failed to trigger, as the initial implementation did not consider this keypress as a space, or as a change to the context.

> {productname} 6.8 addresses this issue, now, the **Spell Checker Pro** mechanism is activated on keypress **Enter**, **Spacebar**, **Shift+Enter** or changing context.

Reviewer: To be specific, previously it was triggered on enter **because** it changes the context, not really by pressing enter. Now enter actually triggers it.
It's basically gone from "Triggered by space and context change ( which enter triggers )" to "Triggered by space, enter, and context change ( which enter still triggers, but we don't run it twice )"
For most practical purposes it's gone from "Triggered by space and enter" to "Triggered by space, enter, and shift+enter" though.
I don't know how technical we need to be in this documentation, I leave it up to you.

Author: ```suggestion
{productname} 6.8 addresses this issue, now, the **Spell Checker Pro** mechanism is triggered by either **Space**, **Enter** or **Shift+Enter**.
```
